packages/cozy-harvest-lib/src/components/TriggerManager.jsx

```diff
@@ -226,10 +299,38 @@ export class TriggerManager extends Component {
     if (typeof onError === 'function') onError(error)
   }
 
-  handleCipherSelect() {
-    this.setState({
-      step: 'accountForm'
-    })
+  handleCipherSelect(selectedCipher) {
+    const { konnector } = this.props
+    const { fields } = konnector
+    const sanitizedFields = manifest.sanitizeFields(fields)
+    const account = this.cipherToAccount(selectedCipher)
+    const values = manifest.getDefaultedValues(konnector, account)
+
+    const requiredFields = Object.entries(sanitizedFields)
+      .filter(([, value]) => value.required)
+      .map(([key]) => key)
+
+    const requiredFieldsInValues = intersection(
```

Comment:
How about `const hasValuesForRequiredFields = intersction().length === requiredFields.length`, and then use `hasValuesForRequiredFields` in the condition below?